tests/run/worktree/data/testplan.fmf

```diff
@@ -0,0 +1,8 @@
+prepare:
+    script: echo CHECK > testfile
```

Comment:
Idea for prepare test extension: `script: grep PASS existing_file` and create existing_file in next to testplan.fmf So we assert you  can find custom script files in prepare step.

Reply:
make sense, thx

Reply:
I believe resolved